aspnetcore/blazor/security/server/index.md

```diff
@@ -28,6 +28,15 @@ Blazor differs from a traditional server-rendered web apps that make new HTTP re
 > [!NOTE]
 > The code examples in this article adopt [nullable reference types (NRTs) and .NET compiler null-state static analysis](xref:migration/50-to-60#nullable-reference-types-nrts-and-net-compiler-null-state-static-analysis), which are supported in ASP.NET Core in .NET 6 or later. When targeting ASP.NET Core 5.0 or earlier, remove the null type designation (`?`) from the examples in this article.
 
+## Server-side security of sensitive data and credentials
+
+In test/staging and production environments, server-side Blazor code and web APIs should use secure authentication flows that avoid maintaining credentials within project code, configuration files, and environment variables. For more information, see the following resources:
```

Comment:
Javier and Steve approved env variables for test. Although it's easy to migrate test configuration to production so that seems like a risk to me.

Reply:
I'll see if DR is willing to have our prohibition against using config files and env vars only apply to production. Right now, I only have those recommended for local development testing.

Reply:
> I'll see if DR is willing to have our prohibition against using config files and env vars only apply to production. Right now, I only have those recommended for local development testing.

Config files for sure shouldn't be used. Env for test is risky if you ask me because customers are very likely to forget and use them in production. It's just too easy to promote test to production.